### PR TITLE
Add wrapper post around David Seddon's atomic article

### PR DESCRIPTION
**Pull request by codeinthehole**
image

### CODE REVIEW

File: _posts/2020-11-30-durable-transactions.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[David Seddon](https://twitter.com/seddonym), a senior developer in our team,
+recently published an interested blog post on ["The trouble with
+`transaction.atomic`"](https://seddonym.me/2020/11/19/trouble-atomic/) drawing
+on internal discussions within our tech team.
+
+This was picked up by the core Django team and a new `durable` flag has already
+been added to the `django.db.transaction.atomic` function — see the [development
+branch docs](https://docs.djangoproject.com/en/dev/topics/db/transactions/#django.db.transaction.atomic).
+
+We can look forward to this feature in Django 3.2.
```
Comment (codeinthehole) [inline _posts/2020-11-30-durable-transactions.md:19]: @seddonym This accurate?

File: _posts/2020-11-30-durable-transactions.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+category: news
+author: David Winterbottom
+banner: /assets/img/posts/2015-11-23-tech-jobs.jpg
+hex: 0e1720
+---
+
+[David Seddon](https://twitter.com/seddonym), a senior developer in our team,
+recently published an interested blog post on ["The trouble with
+`transaction.atomic`"](https://seddonym.me/2020/11/19/trouble-atomic/) drawing
+on internal discussions within our tech team.
```
Comment (codeinthehole) [inline _posts/2020-11-30-durable-transactions.md:13]: @seddonym Hope you don't mind me linking back to the team?

Comment (seddonym) [reply _posts/2020-11-30-durable-transactions.md:13]: Not at all!

File: _posts/2020-11-30-durable-transactions.md (hunk 1 above)
Comment (seddonym) [reply _posts/2020-11-30-durable-transactions.md:19]: I think so, yes.

**Review by seddonym — APPROVED** [review]
Thanks!

File: _posts/2020-11-30-durable-transactions.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Duration database transactions in Django
```
Comment (timb07) [inline _posts/2020-11-30-durable-transactions.md:2]: "Durable", not "Duration", surely?

**Pull request merged**